cppfd/CMakeLists.txt

```diff
@@ -17,15 +19,19 @@ if (ccache_binary)
   set_property(GLOBAL PROPERTY RULE_LAUNCH_COMPILE "${ccache_binary}")
 endif()
 
+configure_file(cfd_config.h.in cfd_config.h)
+
 # Add library targets
 add_library(mesh_chunk mesh_chunk.cpp)
 add_library(parallel_mesh parallel_mesh.cpp)
 add_library(boundary_conditions boundary_conditions.cpp)
 add_library(solver solver.cpp)
 
 # Add executable target
-add_executable(test test.cpp)
-add_executable(parallel_test parallel_test.cpp)
+if(OUTPUT_VTK_FILES)
+  add_executable(test test.cpp)
```

Comment:
These tests must be compiled even when `OUTPUT_VTK_FILES` is OFF -- it's just that the VTK files should not be output. But aside from the visualization files output, the tests should be compiled still.